airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py

```diff
@@ -187,12 +187,17 @@ def extract_image_pull_secrets(pod, req):
                 'name': pull_secret
             } for pull_secret in pod.image_pull_secrets.split(',')]
 
+    @staticmethod
+    def extract_security_context(pod, req):
+        if pod.security_context is None:
```

Comment:
I prefer this style - rather than having the entire body of the function indented. (The rule I try to follow is to keep things as un-indented as possible.

Reply:
Ah I see! I guess it's a matter of opinion then, I tend to go with "not breaking the code flow", avoiding exceptions in the middle of the code, and impromptu returns so the code can be followed top to bottom

If that's the style followed in the Airflow project however no problems for me :)

Reply:
Style things often are  :D